HUD/N_0xf83d0febe75e62c9.md

```diff
@@ -19,12 +19,19 @@ See [here](https://docs.fivem.net/docs/game-references/hud-colors/) for the hud
 
 Name in scripts: `SETUP_FAKE_CONE_DATA()`
 
+For people who don't know how to convert degrees in radians and the other way around:
+
+```
+degrees = radians * 180 / pi
+radians = degrees * pi / 180
+```
+
 ## Parameters
 * **blip**: A blip from an entity, for example ch_prop_ch_cctv_cam_01a
 * **p1**: -1.0 in decompiled scripts (unk)
 * **p2**: 1.0 in decompiled scripts (unk)
 * **width**: Width of the cone (0.0 to ~3.0) 
 * **p4**: 1.0 in decompiled scripts (unk)
 * **length**: Length of the cone. (8.2 is 'normal' sized
-* **heading**: Not your typical heading. Can be calculated with (heading + 180.00) * 0.017453292
+* **heading**: Not your typical heading. This is in radians. Can be calculated with (heading + 180.00) * (pi/180)
```

Comment:
"This heading is in radians, this can be calculated with (heading + 180.00) * (pi/180)" might be better wording

Reply:
welp this is one of my weird habits of doing _exactly_ what is being told to me and dropping my own 'quality control' :/